classes/Hook.php

```diff
@@ -683,6 +683,8 @@ public static function exec($hook_name, $hook_args = array(), $id_module = null,
                     $display = Hook::coreCallHook($moduleInstance, 'hook'.$hook_name, $hook_args);
                 } elseif ($hook_retro_callable) {
                     $display = Hook::coreCallHook($moduleInstance, 'hook'.$retro_hook_name, $hook_args);
+                } else {
+                    continue;
```

Comment:
For which line ? Actually, if cannot go inside `if ($hook_callable)`or `elseif ($hook_retro_callable)`, we won't have any issue thanks to the line 674 which do both tests before entering this block:
```php
if ($hook_callable || $hook_retro_callable)
```

This won't throw any error of course, but in my opinion this code is unreachable.